addons/osfstorage/tests/test_models.py

```diff
@@ -731,16 +736,13 @@ def test_fork_reverts_to_using_user_storage_default(self):
         child_settings.save()
 
         fork = project.fork_node(Auth(user))
-        child_fork = models.Node.objects.get_children(fork).first()
-        assert fork.get_addon('osfstorage').region_id == us.id
-        assert fork.get_addon('osfstorage').user_settings == user.get_addon('osfstorage')
-        assert child_fork.get_addon('osfstorage').region_id == us.id
+        fork_region_id = fork.get_addon('osfstorage').region_id
 
-        fork = project.fork_node(Auth(user2))
```

Comment:
this should test that users from different regions get the same fork results, but right now it only tests that `user` (with `region: us`) gets the correct results, and `user2` (with `region: canada`) is not used at all.

maybe add a `fork2 = project.fork_node(Auth(user2))` and check that it's also correct?

Reply:
I just decided to re-write this whole test class in pytest to make it more explicit. Hopefully this covers the correct case.